reason-react-native/src/components/FlatList.re

```diff
@@ -18,19 +18,15 @@ external make:
     ~getItemLayout: (array('item), int) => VirtualizedList.itemLayout=?,
     ~numColumns: 'int=?,
     // VirtualizedList props
-    // ~_CellRendererComponent: VirtualizedList.cellRendererComponent('item)=?,
+    ~_CellRendererComponent: VirtualizedList.cellRendererComponent('item)=?,
```

Comment:
I didn't add this since it's not part of FlatList flow types https://github.com/facebook/react-native/blob/de0d7cfb79c7f4011d4b6748b1afc656d33fd5ac/Libraries/Lists/FlatList.js

Reply:
According to both RN documentation how `FlatList` wraps `VirtualList`, `CellRendererComponent` should be a valid prop. However, in looking into implementation of `VirtualList` in RN, there is a comment that says there is a conflict between that and `renderItem`. I don't know what to make of that, but we also have `CellRendererComponent` as a todo in the bindings for `VirtualList`.

Reply:
> we also have CellRendererComponent as a todo in the bindings for VirtualList.

Not sure what you mean.

Reply:
In `VirtualizedList.re`, we have
```
// TODO
type cellRendererComponentProps('item) = {.};
type cellRendererComponent('item) =
  React.component(cellRendererComponentProps('item));
```
I couldn't spend much time to figure out if this is as it should be, but given the TODO comment, I suppose it's worth testing.

Reply:
oh I see. I missed that.
I think we should stick to flow types annotation & not try to make assumption or try the code.
And if flow annotation aren't good, we should maybe try to have those fixed & improve bindings to be in sync.

Reply:
`VirtualizedList.js` in RN has
```
  // TODO: Conflicts with the optional `renderItem` in
  // `VirtualizedSectionList`'s props.
  renderItem: $FlowFixMe<renderItemType>,
```
therefore we can only get so far depending on Flow types. If I can make sense of things, I'll prepare a PR for RN, but I test almost everything anyway.

I also seem to have misremembered the comment, `renderItem` props in `VirtualizedList` and `VirtualizedSectionList` are the ones that are in conflict (and not `CellRendererComponent`), just as `data` in `FlatList` conflicts that from `VirtualizedList`.

Back to `CellRendererComponent`, that one simply seems to have a misleading name. `renderItem` is used to, well render, each item, but the resultant `React.element` is wrapped in `CellRenderer `if the `CellRendererComponent` prop is provided, otherwise in `View`. I was able to successfully use the prop but I'll try a bit more. On the other hand, this seems hardly an essential prop. I could simply define merge `renderItem` and`CellRendererComponent` definitions so that the latter is not needed.